tests/e2e/specs/utils/flinkStatement.ts

```diff
@@ -0,0 +1,125 @@
+import { FrameLocator, Page, expect } from "@playwright/test";
+import { FlinkStatementTestIds } from "./testIds";
+
+/**
+ * Submit a Flink statement to Confluent Cloud.
+ *
+ * @param page - The Playwright page object.
+ * @param fileName - The name of the Flink SQL file to submit. Must be present in the `tests/fixtures` directory.
+ */
+export async function submitFlinkStatement(page: Page, fileName: string) {
+  // First, expand the CCloud env
+  await (await page.getByText("main-test-env")).click();
+
+  // Click on the first Flink compute pool
+  await (await page.getByText("AWS.us-east-1")).click();
```

Comment:
This is the only part that I'm a little concerned about, just in the event we switch accounts and don't have the right resources configured. May be worth pushing some of these expected identifiers out to a separate fixture (or even in the .env?) in a follow-up branch for easier updating later.